R/condition_message_linter.R

```diff
@@ -1,9 +1,39 @@
 #' Block usage of `paste()` and `paste0()` with messaging functions using `...`
 #'
-#' `stop(paste0(...))` is strictly redundant -- `stop(...)` is equivalent.
-#'   `stop(...)` is also preferable to `stop(paste(...))`. The same applies to
-#'   all default condition functions, i.e., [stop()], [warning()], [message()],
-#'   and [packageStartupMessage()].
+#' This is because
```

Comment:
I'd avoid a pronoun (`This`) to start the documentation, instead spell out what `This` is (note that in the actual Rd, this sentence is not connected to the title):

> This linter discourages combining condition functions like [stop()] with string concatenation functions [paste()] and [paste0()]. This is because.

Then below we don't need to link [stop()]